### ISSUE TITLE
Pearson Ranking of 11 Features Image is having Indexes instead of verbiage for each feature.

### ISSUE THREAD

**Issue by bharaniabhishek123** [DistrictDataLabs/yellowbrick#404]
[Short Description]
Pearson Ranking of 11 Features Image is having Indexes instead of verbiage for each feature.
### Proposal/Issue
Image should have name of features on X and Y axis.
[Describe expected vs actual behavior or feature proposal]
Actual : Index Numbers
Expected : Name of Feature
### Code Snippet
visualizer = Rank2D(algorithm="pearson")
visualizer.fit_transform(X)
visualizer.poof()
[Code to produce bug or example code for visualizer]

### Background

[Optional, add links and background on the feature or problem]
<img width="520" alt="screen shot 2018-05-14 at 11 19 27 am" src="https://user-images.githubusercontent.com/17970177/40006696-0a539648-5769-11e8-9641-9a614f9eec61.png">

**Comment by jQwotos** [comment]
@bharaniabhishek123 what dataset are you using? Can't seem to reproduce on multiple datasets (such as [Honey Production dataset](https://www.kaggle.com/jessicali9530/honey-production/data) )

I also believe that it should be
```
visualizer = Rank2D(features=[your list of features], algorithm="pearson")
```

**Comment by bharaniabhishek123** [comment]
it's an issue with documentation ,nothing wrong with code.

**Comment by bharaniabhishek123** [comment]
Fixed and created a pull request.
